.vscode/settings.json

```diff
@@ -61,13 +61,13 @@
   "python.pythonPath": "${workspaceFolder}/.venv/bin/python",
   // test discovery is sluggish and the UI around running
   // tests is often in your way and misclicked
-  "python.testing.pyTestEnabled": false,
+  "python.testing.pytestEnabled": false,
   "python.testing.unittestEnabled": false,
   "python.testing.nosetestsEnabled": false,
   "editor.tabSize": 4,
   "restructuredtext.confPath": "",
   "python.linting.enabled": true,
   "python.linting.flake8Path": "${workspaceFolder}/.venv/bin/flake8",
   "python.linting.pep8Path": "${workspaceFolder}/.venv/bin/pep8",
-  "python.testing.pyTestPath": "${workspaceFolder}/.venv/bin/pytest"
+  "python.testing.pytestPath": "${workspaceFolder}/.venv/bin/pytest"
```

Comment:
@MeredithAnya any reason you are changing the vscode settings?

Reply:
ugh I need to remove this, I dk why this keeps getting changed